### PR TITLE
Add example.md

### PR DESCRIPTION
**Pull request by janus**
Add examples and also how to explore eth-cli.

### CODE REVIEW

**Comment by fvictorio** [comment]
This has a lot of duplication with the readme, doesn't it?

**Comment by janus** [comment]
@fvictorio ,

Some of the readme examples are great .. should I remove all the duplication?

File: example.md [hunk 1]
```diff
@@ -0,0 +1,590 @@
...
+
+
+## Examples
+
+There are a lot of things that you can do with `eth-cli`, and we keep adding more. These are some of our favorites:
+
+
+### Use help to learn how to play around with eth-cli
+
+```
```
Comment (ChrisChinchilla) [inline example.md]: @janus How do you get this output?

File: example.md [hunk 2]
```diff
@@ -0,0 +1,590 @@
...
+'20000000000'
+
+```
+
+```
+mainnet> web3.eth.getCode("0xD29F0b5b3F50b07Fe9a9511F7d86F4f4bAc3f8c4").then(console.log);
+'0x6080604052600436106100af576000357c010000000000000000000000000000...
+```
+
+You can also preload some contracts using a `<abi>@<address>` syntax:
```
Comment (ChrisChinchilla) [inline example.md]:
```suggestion
You can also preload contracts using a `<abi>@<address>` syntax:
```

File: example.md [hunk 3]
```diff
@@ -0,0 +1,590 @@
...
+$ eth ra
+{
+  "address": "0x0e1c447DE1e870b5056e596C80e34da26569Deb2",
+  "privateKey": "0x9093070def746fe7a3ad548b10dc0e6f861dc8753a41cd062de32c298517d4eb"
+}
+```
+
+### Get info about the most popular chains
+
+Do you keep forgetting what is the id of rinkeby? Just run `eth networks
```
Comment (ChrisChinchilla) [inline example.md]:
```suggestion
Do you keep forgetting the id for the rinkeby network? Just run `eth networks
```

File: example.md [hunk 4]
```diff
@@ -0,0 +1,590 @@
...
+
+```
+eth repl ./abis/myContract.json@0xaD2FA57bd95A3dfF0e1728686997F6f2fE67F6f9
+eth repl erc20@0x34ee482D419229dAad23f27C44B82075B9418D31 
+```
+
+
+## Installation
+
+Install it globally:
```
Comment (ChrisChinchilla) [inline example.md]:
```suggestion
Install eth-cli globally:
```

File: example.md [hunk 5]
```diff
@@ -0,0 +1,590 @@
...
+
+`npm install -g eth-cli`
+
+You can also try it out with `npx`:
```
Comment (ChrisChinchilla) [inline example.md]:
```suggestion
You can also try installing eth-cli with `npx`:
```

File: example.md [hunk 6]
```diff
@@ -0,0 +1,590 @@
...
+```
+$ eth tx:get --kovan 0x7f3f2967b93a7eded18071a596b7f3987c9575b6bae0a19399725c19e793517b
+{
+  "blockHash": "0xc85b64cdb940da308058b1f0ae800954a09287e2b8d30e71f02e1a4cf5b1a03b",
+  "blockNumber": 11365979,
+  "chainId": "0x2a",
+  ...
+```
+
+The transaction object will also include a `receipt` key, that will have the
```
Comment (ChrisChinchilla) [inline example.md]:
```suggestion
The transaction object also includes a `receipt` key, that has the
```

**Comment by ChrisChinchilla** [comment]
Some small changes @janus , but yes, there is a bit of duplication. Any guidance on what to keep and remove @fvictorio ?

File: example.md (hunk 1 above)
Comment (janus) [reply example.md]: Please , which output are you referring?

**Comment by fvictorio** [comment]
I think there are two options here: either make this file become the new `README`, or to remove everything but the examples section, and link to this file from the readme (but I'd still like to have some examples in the readme and in any case to link to a doc with more in depth examples).

**Pull request closed without merging**